test/junit/scala/OptionTest.scala

```diff
@@ -11,6 +11,14 @@ class OptionTest extends RunTesting {
   @Test def testNoneZipSome: Unit = assertEquals(None zip Some("bar"), None)
   @Test def testNoneZipNone: Unit = assertEquals(None zip None, None)
 
+  @Test def testSomeUnzipToSomePair: Unit = assertEquals(Some(("foo", "bar")) unzip, (Some("foo"), Some("bar")))
+  @Test def testSomeUnzipToSomeNone: Unit = assertEquals(Some(("foo", null)) unzip, (Some("foo"), Some(null)))
+  @Test def testNoneUnzipToNonePair: Unit = assertEquals(None unzip, (None, None))
+
+  @Test def testSomeUnzip3ToSomeTriple: Unit = assertEquals(Some(("foo", "bar", "z")) unzip3, (Some("foo"), Some("bar"), Some("z")))
+  @Test def testSomeUnzip3ToSomeNone: Unit = assertEquals(Some(("foo", null, null)) unzip3, (Some("foo"), Some(null), Some(null)))
+  @Test def testNoneUnzip3ToNoneTriple: Unit = assertEquals(None unzip3, (None, None, None))
+
```

Comment:
we generally avoid postfix syntax - it would be better to have `.unzip` and `.unzip3`